Application Settings/StudentRegestrationController.cls

```diff
@@ -0,0 +1,28 @@
+public with sharing class StudentRegestrationController {
+    public Student__c studentInstance{get;set;}
+    public String languageOption{get;set;}
+    public List<SelectOption> language;
+    
+    // Creating an instance in constructor
+    public StudentRegestrationController(ApexPages.StandardController StandardControllerExtension) {
+        studentInstance = (Student__c)StandardControllerExtension.getRecord();
+    }
+    
+    // Add the languages to the picklist     
+    public List<SelectOption> getlanguage() {
```

Comment:
method name should be getAllLanguages -> and these languages should come from custom settings